### PR TITLE
added the blog page ui

### PR DESCRIPTION
**Pull request by c-blaq**

### CODE REVIEW

File: articles/top-coding-careers.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 title: Top 3 Coding Careers to pursue in 2021
 author: John Doe
-date: 01-7-2021
+date: 01-7-2021 4mins
```
Comment (funabab) [inline articles/top-coding-careers.md]: This isn't a valid datetime string. Read more about valid date string format here.

https://en.m.wikipedia.org/wiki/ISO_8601

File: pages/blog/[slug].jsx [hunk 2]
```diff
@@ -31,6 +37,7 @@ const PostPage = ({
   author,
   authorImage,
   excerpt,
+  date,
```
Comment (funabab) [inline pages/blog/[slug].jsx]: The there is no date prop being passed to the component. On a time prop, which is a time stamp is being passed.

File: pages/blog/[slug].jsx [hunk 3]
```diff
@@ -40,23 +47,66 @@ const PostPage = ({
...
-          <Link href={`/tag/${tag}`}>
-            <a>#{tag}</a>
-          </Link>{" "}
-          &nbsp;
-        </React.Fragment>
-      ))}
+      <Blog>
+        {coverImage && (
+          <div className="blog--cover-image">
+            {<img src={coverImage} alt="Cover image" />}
```
Comment (funabab) [inline pages/blog/[slug].jsx]: You don't need to wrap this around curly braces

Comment (c-blaq) [reply pages/blog/[slug].jsx]: Noted

File: pages/blog/[slug].jsx (hunk 2 above)
Comment (c-blaq) [reply pages/blog/[slug].jsx]: Oh yes, I used it for testing. Will remove it

File: articles/top-coding-careers.md (hunk 1 above)
Comment (c-blaq) [reply articles/top-coding-careers.md]: Noted

**Pull request merged**